### ISSUE TITLE
fails with https://github.com/strongloop/loopback-example-full-stack example

### ISSUE THREAD

**Issue by RainerAtSpirit** [SimonDegraeve/loopback-webpack-plugin#1]
I was trying to get this plugin working with the ootb loopback full stack example, but it throws a bunch of error messages.
This might be related to the fact that I'm testing on Win 8.1, though.
Is there a working repo that I could use as starting point?

**Comment by Nutelac** [comment]
Same problem. I've got bunch of `Cannot resolve module [fs, net, browserify, ...]` errors.

**Comment by lukasredev** [comment]
Same problem here.

**Comment by Nutelac** [comment]
I've created issue in Loopback repo https://github.com/strongloop/loopback/issues/1412.

**Comment by rob3c** [comment]
It looks like that loopback-example-full-stack doesn't exist anymore, at least by that name.

Unfortunately, `loopback-webpack-plugin` also fails with the other two browserify-based strongloop samples I've tried:
  https://github.com/strongloop/loopback-example-offline-sync
and
  https://github.com/strongloop/loopback-example-isomorphic

The loopback webpack issue mentioned above is still open. Anyone get webpack + loopback working yet?

**Comment by zamb3zi** [comment]
Yes, I've documented my solution here: [`loopback-webpack-example`](https://github.com/zamb3zi/loopback-webpack-example)
